_posts/2022-04-29-monero-GUI-0.17.3.2-released.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "Monero GUI 0.17.3.2 'Oxygen Orion' released"
+summary: Point release containing support for P2Pool mining and Ledger Nano S Plus.
+tags: [releases]
+author: selsta
+image: /blog/assets/2020-09-17-monero-0.17-released/oxygen-orion.png
+---
+
+<div align="center">
+    <img src="{{ page.image }}" width="350px">
+</div>
+
+# Overview
+
+This is the v0.17.3.2 point release of the Monero GUI software. This release contains support for P2Pool mining and Ledger Nano S Plus.
+
+[The latest CLI release notes can be found on the precedent blog post]({{ site.baseurl_root }}/2022/04/29/monero-0.17.3.2-released.html)
+
+Some highlights of this release are:
+
+- Integrate experimental support for P2Pool mining (#[3829](https://github.com/monero-project/monero-gui/pull/3829))
+- Warn against high fees during transaction creation (#[3897](https://github.com/monero-project/monero-gui/pull/3897))
+- Improvements against wallet getting stuck on exit (#[3890](https://github.com/monero-project/monero-gui/pull/3890), #[3889](https://github.com/monero-project/monero-gui/pull/3889))
+- Add support for Ledger Nano S Plus
+- Add support for reserve proof (#[3828](https://github.com/monero-project/monero-gui/pull/3828))
+- Add a lock wallet button to the title bar (#[3859](https://github.com/monero-project/monero-gui/pull/3859))
+- Fix offline signing (#[3862](https://github.com/monero-project/monero-gui/pull/3862))
+- Fix adding a new address book entry (#[3865](https://github.com/monero-project/monero-gui/pull/3865))
+- Update translations, add support for 5 new languages (#[3832](https://github.com/monero-project/monero-gui/pull/3832))
+- Minor bug fixes and UI improvements
+
+The complete list of changes is [available on GitHub](https://github.com/monero-project/monero-gui/compare/v0.17.3.1...v0.17.3.2), along with [the source code](https://github.com/monero-project/monero-gui/tree/v0.17.3.2).
+
+# Contributors for this Release
+
+This release was the direct result of 22 who worked, largely unpaid and altruistically, to put out 76 commits containing 31625 new lines of code. We'd like to thank them very much for their time and effort. In no particular order they are:
+
+- selsta
+- luigi1111
+- rating89us
+- Monero-Weblate
```

Comment:
not a contributor, but you could add @netrik182, who is now managin that account

Reply:
done